Sources/AegithalosCocoa/UIAccessibility.swift

```diff
@@ -0,0 +1,115 @@
+#if canImport(UIKit)
+
+import UIKit
+import Aegithalos
+
+public extension Setup where Subject: UIAccessibilityIdentification {
+
+  @inlinable func accessibilityIdentifier(_ identifier: String?) -> Setup {
+    composed { (subject: Subject) in subject.accessibilityIdentifier = identifier }
+  }
+}
+
+public extension Setup where Subject: UIAccessibilityContentSizeCategoryImageAdjusting {
+
+  @inlinable func adjustsImageSizeForAccessibilityContentSizeCategory(_ adjust: Bool) -> Setup {
+    composed { (subject: Subject) in subject.adjustsImageSizeForAccessibilityContentSizeCategory = adjust }
+  }
+}
+
+public extension Setup where Subject: NSObject {
```

Comment:
Maybe will be better to use UIView?
```suggestion
public extension Setup where Subject: UIView {
```

Reply:
It would limit usage of api. There are elements that don't inherit from UIView e. g. UIBarButtonItem